### PR TITLE
Initial Mainnet config update changes

### PR DESCRIPTION
**Pull request by eddycpc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "@babel/helper-builder-react-jsx": "^7.10.4",
     "@babel/helper-builder-react-jsx-experimental": "^7.12.11",
     "@cosmjs/stargate": "^0.24.0-alpha.22",
-    "@crypto-com/chain-jslib": "^0.0.10",
+    "@crypto-com/chain-jslib": "^0.0.12",
```
Comment (Nawaz-Malik143) [inline package.json:16]: Thanks

**Review by Nawaz-Malik143 — APPROVED** [review]

**Comment by calvinaco** [comment]
I remember there is a warning popup when using the Mainnet, is that warning removed? Thanks @eddycpc

**Comment by eddycpc** [comment]
> I remember there is a warning popup when using the Mainnet, is that warning removed? Thanks @eddycpc

The warning pop up only shows if the `nodeURL` is 'TO_BE_DECIDED', now that the node url has a real value, it can't show anymore

**Review by calvinaco — APPROVED** [review]

**Review by Nawaz-Malik143 — APPROVED** [review]

**Review by Nawaz-Malik143 — APPROVED** [review]

**Review by Nawaz-Malik143 — APPROVED** [review]

**Review by Nawaz-Malik143 — APPROVED** [review]

**Review by Nawaz-Malik143 — APPROVED** [review]

**Review by Nawaz-Malik143 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -164,7 +164,7 @@
     "dotenv": "8.2.0",
     "dotenv-expand": "5.1.0",
     "electron": "11.1.1",
-    "electron-builder": "^22.9.1",
```
Comment (Nawaz-Malik143) [inline package.json:167]:
```suggestion
    "electron-builder": "^22.9.1",
```

**Review by Nawaz-Malik143 — APPROVED** [review]

**Review by crypto-matto — APPROVED** [review]

**Review by leejw51crypto — APPROVED** [review]
lgtm

**Review by Nawaz-Malik143 — COMMENTED** [review]
[]()

**Review by Nawaz-Malik143 — COMMENTED** [review]
> **_[]()_**

**Pull request merged**